rllib/core/learner/learner.py

```diff
@@ -1088,6 +1094,9 @@ def update_from_iterator(
                 "`num_iters` instead."
             )
 
+        if not self.iterator:
```

Comment:
Dumb question: What if `self.iterator` is already set (to a previously incoming `DataIterator`)? Would the now-incoming `iterator` be thrown away?

Reply:
Yes, it would. But we want this: The incoming iterator might be different from the one assigned to the learner, but only because of `ray.get` not being able to call learners in order, i.e. the learners might get different streaming splits every training iteration - we do not want this to happen. One Learner same split.